### ISSUE TITLE
printer.feedRows(1) prints 'J' in stead of a row feed, unknown firmware version

### ISSUE THREAD

**Issue by ChrisIdema** [adafruit/Adafruit-Thermal-Printer-Library#38]
`printer.feedRows(1); `prints a J in stead of a row feed. 
I suspected my firmware version is older, however I don't know the firmware version. Setting PRINTER_FIRMWARE in Adafruit_Thermal.h to 0 didn't change the behavior! That is strange, because it should print new-lines in case of older versions that don't support the row-feed command.
I changed the header files in both these locations:

- /home/USER/.arduino15/staging/libraries/Adafruit_Thermal_Printer_Library-1.2.0.zip
- /home/USER/Arduino/libraries/Adafruit_Thermal_Printer_Library

I added the following lines in my code as a sanity check:
```
#if PRINTER_FIRMWARE >=268
#error "PRINTER_FIRMWARE>=268"
#endif
```
It gives the error when I don't edit the header file and doesn't when I set it to 0. So that means it should be compiled using that define. I deleted the cache folder and restarted arduino ide several times. But it still prints J

I use printer #597.
This is what the sticker says:
```
Model: CSN-A2
S/N Z1912110420
```
The test page printed by the printer doesn't have a firmware version number. This is what I see:
```
Selftest
System: A2
Version: GV.16.08 20190822_R
[...]
MAX dots:80
Heat time:774
[...]
Selftest Finished
```
I have ubuntu 20.
I use esp32 DOIT devkit.
Printing text works well.
Printing bitmaps works well too, but I do see thin horizontal white lines through the image even after setting heating time to 255.

**Comment by xMase** [comment]
the problem is this [command](https://github.com/adafruit/Adafruit-Thermal-Printer-Library/blob/a7dc6aeff1c44d72490dcd1c9fb3c1be1dc5068a/Adafruit_Thermal.cpp#L215) which is not interpreted by our firmware.
it would be good and right if they put the correct datasheet for our firmware.

**Comment by caternuson** [comment]
Does commenting out that command fix the issue? Also not sure how #30 relates to this issue?

**Comment by caternuson** [comment]
Hopefully #46 fixes this. Simply removed that offending command. If that doesn't fix it, please comment and we can reopen.
